lib/Dialect/FIRRTL/Transforms/CheckCombLoops.cpp

```diff
@@ -248,12 +248,12 @@ class DiscoverLoops {
   void preprocess(SmallVector<Value> &worklist) {
     // All the input ports are added to the worklist.
     for (BlockArgument arg : module.getArguments()) {
-      auto argType = arg.getType();
+      auto argType = cast<FIRRTLType>(arg.getType());
```

Comment:
I think BlockArguments / module ports can be non-FIRRTLType's, ports and wires can be foreign I believe.

Curious how/why this is safe, but see that it previously already made this assumption so maybe nevermind :+1: .